test-suite/output/Inductive.out

```diff
@@ -11,3 +11,8 @@ Arguments foo A%type_scope x
 Arguments Foo A%type_scope x
 myprod unit bool
      : Set
+option : Type -> Type
+
+option is template universe polymorphic on option.u0
```

Comment:
Shouldn't it be `option : Type@{option.u0} -> Type`, so that the later reference to `option.u0` makes sense? Or conversely to remove the "on option.u0`?

Reply:
You mean when printing univs is unset we should just say `option is template polymorphic`?

Reply:
Yes, typically, good idea.

Reply:
I actually believe that it was better the other way around but I won't lose my sleep on it either, so that's good enough for me.